modules/ols-custom-resource-definition-file-parameters.adoc

```diff
@@ -0,0 +1,498 @@
+// Module included in the following assemblies:
+
+// * about/ols-configuring-openshift-lightspeed.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ols-custom-resource-definition-file-parameters_{context}"]
+= Lightspeed Custom Resource Definition specification
+
+The following tables provide information about {ols-offical} Custom Resource Definition (CRD) specification 
+
+.{ols-short} CRD parameters for Large Language Model (LLM) providers
+[options="header"]
+[cols="l, 4, 1, 1"]
+|===
+|Parameter |Description |Values |Required
+
+|spec:
+  llm:
+    providers:
+      name:
+|Name of the LLM provider.
+|String
+|yes
+
+|spec:
+  llm:
+    providers:
+      type:
+|LLM provider type.
+|azure_openai, bam, openai, watsonx, rhoai_vllm, rhelai_vllm, fake_provider
+|yes
+
+|spec:
+  llm:
+    providers:
+      url:
+|URL for API provider. The URL must start with `https://`.
+|URL
+|No
+
+|spec:
+  llm:
+    providers:
+      deploymentName:
+|Name of the {azure-openai} deployment.
+|string
+|No
+
+|spec:
+  llm:
+    providers:
+      projectID:
+|The {watsonx} project ID.
+|string
+|No
+
+|spec:
+  llm:
+    providers:
+      credentialsSecretRef:
+|The name of the secret object that stores the API provider credentials
+|string.
+|No
+
+|spec:
+  llm:
+    providers:
+      models:
+|List of models provided by the Large Language Model (LLM) provider.
```

Comment:
List of models from the Large Language Model (LLM) provider that you want OLS to use. Just one is required. @rh-tokeefe

Reply:
done